lib/browser/rpc-server.js

```diff
@@ -210,9 +210,9 @@ const unwrapArgs = function (sender, contextId, args) {
           return rendererFunctions.get(objectId)
         }
 
-        const webContentsId = sender.getId()
```

Comment:
Should we use `sender.id` here given some people might be using `affinity` 🤔 Idk if `remote` even works properly with affinity but this stands out as being an incompat point

Reply:
The purpose of check here is to ensure that message is not sent to the old page, the `WebContents` is guaranteed to be the same one so there is no need to compare the `sender.id`, and messages sent to one `WebContents` won't be sent to other `WebContents` with same `affinity`.

The ID returned by `sender.getId()` is changed when renderer process is restarted, comparing the `processId` has the same behavior with the old code. And it is the reason we used `sender.getId()` instead of `sender.id`.

The check in main process however can not tell when a `WebContents` have multiple pages in it, and it is silently checked in the renderer process:

https://github.com/electron/electron/blob/e90c4abc0d7971f5b64bcff7d25fdbebf88148b7/lib/renderer/api/remote.js#L279-L285